### PR TITLE
apex globals

### PR DESCRIPTION
**Pull request by JohnathanHuggett**

### CODE REVIEW

File: tokens/apex/globals/boxShadow.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (JohnathanHuggett) [inline tokens/apex/globals/boxShadow.json:1]: We need to add focus shadows

File: tokens/apex/globals/font/family.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (JohnathanHuggett) [inline tokens/apex/globals/font/family.json:1]: Is Apex using the same font family as cataylst?

File: tokens/apex/globals/font/weight.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (JohnathanHuggett) [inline tokens/apex/globals/font/weight.json:1]: do we need semibold?

File: tokens/apex/globals/outline.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "global-outline-base": {
+    "value": {
+      "color": "{global-color-focus}",
+      "width": "3px",
```
Comment (JohnathanHuggett) [inline tokens/apex/globals/outline.json:5]: is this the right width?

File: tokens/apex/globals/font/family.json (hunk 2 above)
Comment (pragati-amount) [reply tokens/apex/globals/font/family.json:1]: yes

File: tokens/apex/globals/outline.json (hunk 4 above)
Comment (pragati-amount) [reply tokens/apex/globals/outline.json:5]: yes

File: tokens/$metadata.json [hunk 5]
```diff
@@ -35,6 +35,14 @@
...
     "catalyst/components/modal",
     "catalyst/components/list",
+    "catalyst/components/form/input",
+    "catalyst/components/form/label",
+    "catalyst/components/form/helper",
+    "catalyst/components/form/validation",
+    "catalyst/components/form/verticalSpacing",
+    "catalyst/components/form/select",
+    "catalyst/components/descriptionList",
+    "catalyst/components/heading",
```
Comment (estenp) [inline tokens/$metadata.json:45]: I think I merged this incorrectly

File: tokens/apex/globals/font/weight.json (hunk 3 above)
Comment (pragati-amount) [reply tokens/apex/globals/font/weight.json:1]: just checked yes we need semibold.

File: tokens/apex/globals/boxShadow.json (hunk 1 above)
Comment (pragati-amount) [reply tokens/apex/globals/boxShadow.json:1]: I don't have any apex focus shadow specs right now, is it same as globalShadow-2x

Comment (estenp) [reply tokens/apex/globals/boxShadow.json:1]: These are the shadows for outline in Catalyst. They should just match the Apex focus outline width and color:

```
"global-boxShadow-focus-innerShadow": {
    "value": {
      "x": "0",
      "y": "0",
      "blur": "0",
      "spread": "{global-edgeSize-3px}",
      "color": "{global-color-focus}",
      "type": "innerShadow"
    },
    "type": "boxShadow",
    "description": "This is a global box shadow value to be used only when a box shadow is required to create a focus indicator, e.g. where container is too small for outline, specific border radius workarounds"
  },
  "global-boxShadow-focus-dropShadow": {
    "value": {
      "x": "0",
      "y": "0",
      "blur": "0",
      "spread": "{global-edgeSize-3px}",
      "color": "{global-color-focus}",
      "type": "dropShadow"
    },
    "type": "boxShadow",
    "description": "This is a global box shadow value to be used only when a box shadow is required to create a focus indicator, e.g. where container is too small for outline, specific border radius workarounds"
  }
```

**Review by estenp — APPROVED** [review]

**Pull request merged**